src/documentation/documentationengine.cpp

```diff
@@ -27,13 +27,235 @@
+    constexpr const char* CommandKey = "command";
+
+    // Factory
+    constexpr const char* FactoryTitle = "Asset Components";
+    constexpr const char* MembersKey = "members";
+    constexpr const char* OptionalKey = "optional";
+    constexpr const char* ReferenceKey = "reference";
+    constexpr const char* FoundKey = "found";
+    constexpr const char* RestrictionsKey = "restrictions";
+    constexpr const char* ClassesKey = "classes";
+
+    constexpr const char* OtherName = "Other";
+    constexpr const char* OtherIdentifierName = "other";
+    constexpr const char* propertyOwnerName = "propertyOwner";
+    constexpr const char* categoryName = "category";
+
+    // Properties
+    constexpr const char* SettingsTitle = "Settings";
+    constexpr const char* SceneTitle = "Scene";
+    constexpr const char* PropertiesKeys = "properties";
+    constexpr const char* PropertyOwnersKey = "propertyOwners";
+    constexpr const char* TagsKey = "tags";
+    constexpr const char* UriKey = "uri";
+
+    // Scripting
+    constexpr const char* ScriptingTitle = "Scripting API";
+    constexpr const char* DefaultValueKey = "defaultValue";
+    constexpr const char* ArgumentsKey = "arguments";
+    constexpr const char* ReturnTypeKey = "returnType";
+    constexpr const char* HelpKey = "help";
+    constexpr const char* FileKey = "file";
+    constexpr const char* LineKey = "line";
+    constexpr const char* LibraryKey = "library";
+    constexpr const char* FullNameKey = "fullName";
+    constexpr const char* FunctionsKey = "functions";
+    constexpr const char* SourceLocationKey = "sourceLocation";
+    constexpr const char* OpenSpaceScriptingKey = "openspace";
+
+    // Licenses
+    constexpr const char* LicensesTitle = "Licenses";
+    constexpr const char* ProfileName = "Profile";
+    constexpr const char* AssetsName = "Assets";
+    constexpr const char* LicensesName = "Licenses";
+    constexpr const char* NoLicenseName = "No License";
+    constexpr const char* NoDataName = "";
+
+    constexpr const char* ProfileNameKey = "profileName";
+    constexpr const char* VersionKey = "version";
+    constexpr const char* AuthorKey = "author";
+    constexpr const char* UrlKey = "url";
+    constexpr const char* LicenseKey = "license";
+    constexpr const char* NoLicenseKey = "noLicense";
+    constexpr const char* IdentifiersKey = "identifiers";
+    constexpr const char* PathKey = "path";
+    constexpr const char* AssetKey = "assets";
+    constexpr const char* LicensesKey = "licenses";
+
+    // Keybindings
+    constexpr const char* KeybindingsTitle = "Keybindings";
+    constexpr const char* KeybindingsKey = "keybindings";
+
+    // Events
+    constexpr const char* EventsTitle = "Events";
+    constexpr const char* FiltersKey = "filters";
+    constexpr const char* ActionsKey = "actions";
+
+    nlohmann::json documentationToJson(
+                                         const openspace::documentation::Documentation& d)
+    {
```

Comment:
```suggestion
    nlohmann::json documentationToJson(const openspace::documentation::Documentation& d) {
```

? Looks like it might just fit :)